src/zkevm_specs/public_inputs.py

```diff
@@ -158,36 +171,31 @@ def check_row(
             + FQ(GAS_COST_TX_CALL_DATA_PER_ZERO_BYTE) * is_byte_next_zero
         )
 
+        tx_id_diff = row_next.tx_table.tx_id - row.tx_table.tx_id - one
```

Comment:
Sorry for this confusing naming, your understanding is correct. I've made the change, and I removed the tx_id_constraint as it will always hold because of its form.